performance/performance_tuning.rst

```diff
@@ -241,14 +239,14 @@ There are two configuration parameters which relate to the format in which inter
 
 However, for certain data types like hll or hstore arrays, the cost of serializing and deserializing data is pretty high. In such cases, using binary format for transferring intermediate data can improve query performance due to reduced CPU usage. There are two configuration parameters which can be used to tune this behaviour, citus.binary_master_copy_format and citus.binary_worker_copy_format. Enabling the former uses binary format to transfer intermediate query results from the workers to the coordinator while the latter is useful in queries which require dynamic shuffling of intermediate data between workers.
 
-Real Time Executor
+Adaptive Executor
 -------------------------------
 
-If you have SELECT queries which require sub-second response times, you should try to use the real-time executor.
+Adaptive executor is the default executor, and it can handle all workloads very efficiently.
 
-The real-time executor opens one connection and uses two file descriptors per unpruned shard (Unrelated shards are pruned away during planning). Due to this, the executor may need to open more connections than max_connections or use more file descriptors than max_files_per_process if the query hits a high number of shards.
+The adaptive executor is smart enough to use as less connections as possible to utilize minimal resources on the worker nodes for multi-shard queries. The executor starts using a single connection to a remote node, which is almost always cached so it doesn't incur any additional overhead. If the query execution finishes in citus.executor_slow_start_interval, we're done, no more connections are opened. Else, the executor keeps establishing new connections in every citus.executor_slow_start_interval.
```

Comment:
How about this:

The adaptive executor conserves database connections to help reduce resource usage on worker nodes during multi-shard queries. When running a query, the executor begins by using a single -- usually precached -- connection to a remote node. If the query finishes within the time period specified by ``citus.executor_slow_start_interval``, no more connections are required.  Otherwise the executor gradually establishes new connections as directed by ``citus.executor_slow_start_interval``.

Reply:
Also we could link to the [adaptive executor configuration](https://docs.citusdata.com/en/v9.2/develop/api_guc.html#adaptive-executor-configuration) section that has lots of details.

Reply:
done